### PR TITLE
docs(manage): update instance settings

### PR DESCRIPTION
**Pull request by mffap**
```[tasklist]
### Definition of Ready
- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.
```

### CODE REVIEW

File: docs/docs/guides/manage/console/instance-settings.mdx [hunk 1]
```diff
@@ -87,11 +87,14 @@ The Login Policy defines how the login process should look like and which authen
...
+| Register allowed          | Enable self register possibility in the login ui, this enables username password registration as well as registration with configured external identity providers   |
 | External IDP allowed      | Possibility to login with an external identity (e.g Google, Microsoft, Apple, etc), If you like to allow external Identity providers add them to the providers list |
-| Force MFA                 | Force a user to register and use a multifactor authentication, Ensure that you have added the MFA methods you want to allow.                                        |
-| Passwordless              | Choose if passwordless login is allowed or not                                                                                                                      |
+| Hide password reset | Disable the self-service option for users to reset their password. |
+| Domain discovery allowed |  |
+| Ignore unknown usernames |  |
+| Disable login with email address |  |
+| Disable login with phone number |  |
```
Comment (mffap) [inline docs/docs/guides/manage/console/instance-settings.mdx:97]: @hifabienne can you please explain what these options do? Just from the labels its not really clear when to use them.

File: docs/docs/guides/manage/console/instance-settings.mdx [hunk 2]
```diff
@@ -87,11 +87,14 @@ The Login Policy defines how the login process should look like and which authen
...
 | Username Password allowed | Possibility to login with username and password. If this is disabled only login with external identity providers will be allowed                                    |
+| Register allowed          | Enable self register possibility in the login ui, this enables username password registration as well as registration with configured external identity providers   |
 | External IDP allowed      | Possibility to login with an external identity (e.g Google, Microsoft, Apple, etc), If you like to allow external Identity providers add them to the providers list |
-| Force MFA                 | Force a user to register and use a multifactor authentication, Ensure that you have added the MFA methods you want to allow.                                        |
-| Passwordless              | Choose if passwordless login is allowed or not                                                                                                                      |
+| Hide password reset | Disable the self-service option for users to reset their password. |
+| Domain discovery allowed |  |
+| Ignore unknown usernames |  |
```
Comment (mffap) [inline docs/docs/guides/manage/console/instance-settings.mdx:95]: @hifabienne can you please explain what these options do? Just from the labels its not really clear when to use them.

**Comment by mffap** [comment]
@hifabienne would it be worth adding a small guide on what domain discovery is and you can use it in zitadel? Just had the question in a call and we don't seem to have docs on the settings and the use case.

File: docs/docs/guides/manage/console/instance-settings.mdx [hunk 3]
```diff
@@ -87,11 +87,14 @@ The Login Policy defines how the login process should look like and which authen
...
 | Username Password allowed | Possibility to login with username and password. If this is disabled only login with external identity providers will be allowed                                    |
+| Register allowed          | Enable self register possibility in the login ui, this enables username password registration as well as registration with configured external identity providers   |
 | External IDP allowed      | Possibility to login with an external identity (e.g Google, Microsoft, Apple, etc), If you like to allow external Identity providers add them to the providers list |
-| Force MFA                 | Force a user to register and use a multifactor authentication, Ensure that you have added the MFA methods you want to allow.                                        |
-| Passwordless              | Choose if passwordless login is allowed or not                                                                                                                      |
+| Hide password reset | Disable the self-service option for users to reset their password. |
+| Domain discovery allowed |  |
```
Comment (hifabienne) [inline docs/docs/guides/manage/console/instance-settings.mdx:94]:
```suggestion
| Domain discovery allowed | If this setting is enabled, the user does't not mandatory have to exist when entering the username. It is required to have verified domains on the organization. Example: ZITADEL is registered as organization with the domain zitadel.com and AzureAD as identity provider. A user enters john@zitadel.com in the login but the user doesn't exist. The domain can be mapped to the organization and therefore the user can be redirected to the AzureAD.
```

File: docs/docs/guides/manage/console/instance-settings.mdx (hunk 2 above)
Comment (hifabienne) [inline docs/docs/guides/manage/console/instance-settings.mdx:95]:
```suggestion
| Ignore unknown usernames | This setting can be enabled, if no error message should be shown if the user doesn't exist. Example: A user enters the login name john@zitadel.com, the user doesn't exist, but will be redirected to the password screen. After entering a password, the user will get an error that either username or password are wrong. |
```

**Review by hifabienne — APPROVED** [review]

**Pull request merged**